example-llm-workflows/README.md

```diff
@@ -0,0 +1,483 @@
+    let reduce_step = "Summarize facts of the city into a paragraph:\n{{text}}";
+    prompt_chain(&system, &input, map_step, reduce_step, None)
+}
+```
+
+And, here's an example response provided by the model from the executed workflow:
+
+>  Sentence:
+
+> Additionally, the city hosts the annual Three Rivers Arts Festival, showcasing local artists' work and attracting thousands of attendees each year. Sentence: Pittsburgh is also home to over 446 bridges, earning it the nickname "City of Bridges." Sentence: Its rich history and cultural attractions make Pittsburgh a popular destination for both locals and tourists alike.
+
+> Final paragraph:
+
+> Pittsburgh, a city with a strong sense of identity, is more than just steel production and industrial heritage. This iconic city is also home to the famous Andy Warhol Museum, established in 1994, which offers visitors an unparalleled look into the life and artistic vision of the legendary artist. In addition to its cultural attractions, Pittsburgh hosts the annual Three Rivers Arts Festival, a celebration of local artistry that draws thousands each year. Furthermore, the city's impressive record of over 446 bridges has earned it the nickname "City of Bridges." With its rich history and cultural offerings, Pittsburgh is a beloved destination for both locals and visitors seeking to experience the best of this remarkable city. Overall, Pittsburgh's unique blend of artistic expression, industrial history, and stunning architecture makes it a must-see place that leaves a lasting impression on those who visit.
+
+### Idempotent Replay
+
+Typically when relaying questions and prompts to LLM applications, you'll get varied responses and answers even when you don't change the input text.
+
+When a Homestar node receives a workflow containing a task that has already been executed, either with its index (key) and output (value) stored locally (most optimal) or retrieved from a shared DHT (Distributed Hash Table) over the network. In that case, it can replay that task without re-execution or having to fetch the Wasm module from its source. This replayability is akin to [durable function/execution][durable-fn] frameworks like [Cadence][cadence] and [Temporal][temporal] but happens over a decentralized mesh instead of a centralized infrastructure.
+
+So, what does that mean for us? If we provide the same arguments to our Guest Wasm functions, Homestar will return cached responses instantly, which saves us both time and resources, as waiting for large prompt responses when using models, even *smaller* ones like LLaMA 3 trained with 8 billion parameters, can be somewhat latent on devices with limited GPU and memory (like my personal computer).
+
+Is there a way to always provide *new* hallucinations from the Host model? Yes, there is! We can use the `--debug` flag when we start EveryCli to force re-execution of the tasks in our workflow.
```

Comment:
Nice! This is a great lever maintaining consistency or just exploring responses.